### ISSUE TITLE
HTTP2 SETTINGS iOS support

### ISSUE THREAD

**Issue by Aron148814** [sleeyax/burp-awesome-tls#58]
Will there be support for iOS?
Because iOS devices only use MaxConcurrentStreams and InitialWindowSize.

![Screenshot at Jan 25 14-36-46](https://github.com/sleeyax/burp-awesome-tls/assets/157375521/a6916644-3dc9-44b9-9370-89ab9ae8e11d)

**Comment by sleeyax** [comment]
I don't have an iOS device to test this myself at the moment so I would happily accept a PR for this feature!

**Comment by Aron148814** [comment]
Connecting iPhone with Burp Suite (without extension) changes the http2 fingerprint already with SETTINGS and WINDOW_UPDATE Frame.
<img src="https://github.com/sleeyax/burp-awesome-tls/assets/157375521/ebf5b956-14f6-4795-93bf-c704794ee2c1" height="300"> <img src="https://github.com/sleeyax/burp-awesome-tls/assets/157375521/31bb352e-af38-4e0d-80ac-609f147aa84b" height="300">
So Its also the WINDOW_UPDATE Frame and when I just use InitialWindowSize and MaxConcurrentStreams In the source code I get "bad record MAC" In the server console.

**Comment by Aron148814** [comment]
I fixed It by modifying some code in h2_bundle In oohttp also the same for WINDOW_UPDATE Frame.

**Comment by sleeyax** [comment]
Can you submit a pr to my fork?

**Comment by Aron148814** [comment]
Yea I will do It in couple of days and this Is what I added in the transport.go file.

![tls](https://github.com/sleeyax/burp-awesome-tls/assets/157375521/89a0845d-d3ee-442e-ae9a-b31bea6d0da7)

**Comment by sleeyax** [comment]
I'll look forward to you PR, I can't comment on a screenshot.

**Comment by Aron148814** [comment]
Do you know anything about the Priority flag In the HEADERS Frame?
Because I get priority flag on real device (no burp) and It only happens on the site "tls.peet.ws/api/all" not on browserleaks or browserscan.
Its not that important because it doesn't change the HTTP/2 Fingerprint but Its still weird.
![httpheadercompare](https://github.com/sleeyax/burp-awesome-tls/assets/157375521/8e9aa425-d370-442a-bdaf-34f151ea61dc)

**Comment by sleeyax** [comment]
@Aron148814 have you figured it out by now? I would still be happy to accept a PR for your changes if you can.

**Comment by Aron148814** [comment]
@sleeyax Yeah sorry Im kinda been busy this last month but I will try this month for sure.

**Comment by sleeyax** [comment]
Any news @Aron148814 ?

**Comment by Aron148814** [comment]
@sleeyax You didn't saw my pull request on your forked oohttp repository?

**Comment by sleeyax** [comment]
O.o

No... My bad let me check and review in a moment.

**Comment by sleeyax** [comment]
Added in https://github.com/sleeyax/burp-awesome-tls/releases/tag/v2.0.0.
